source/contracts/extensions/MarketFeeCalculator.sol

```diff
@@ -13,11 +13,13 @@ contract MarketFeeCalculator {
     mapping (address => uint256) private shareSettlementPerEthFee;
     mapping (address => uint256) private validityBondInAttoeth;
     mapping (address => uint256) private targetReporterGasCosts;
+    mapping (address => uint256) private designatedReportStakeInAttoRep;
```

Comment:
All of these should probably be moved to be stored in a non-extension contract like Universe.  The values should _not_ carry across universes, and things in Extensions are meant to be "helpers", not storing critical pieces of the system.  We may want to consider moving this contract out of extensions and into the core.  If we do keep data in here (it still needs to be indexed by Universe) we should probably delegate it.

My recommendation is to move this contract into the core suite of contracts and get these storage items moved over to Universe.

Reply:
Feel free to create a clubhouse issue and do the above in a separate PR.

Reply:
Will do! I had similar thoughts about this contract since it's storing/writing data and would need to be audited.